src/js/components/Button/BusyAnimation.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import styled, { css, keyframes } from 'styled-components';
+import { FormCheckmark } from 'grommet-icons/icons/FormCheckmark';
+import { Box } from '../Box';
+
+const bounceDelay = keyframes`
+  0%, 80%, 100% { 
+    transform: scale(0.4);
+  } 40% { 
+    transform: scale(0.8);
+  }
+`;
+
+const bounceDelayRule = css`
+  animation: ${bounceDelay} 1.4s infinite ease-in-out both;
+`;
+
+const Dot = styled(Box)`
+  width: 8px;
+  height: 8px;
+  background-color: ;
+  ${(props) => props.color && `background-color: ${props.color};`}
+  border-radius: 100%;
+  display: inline-block;
+  ${bounceDelayRule}
+  ${(props) => props.delay && `animation-delay: ${props.delay};`}
+`;
+
+export const EllipsisAnimation = ({ color }) => (
+  <Box
+    style={{ position: 'absolute' }}
+    fill
+    alignContent="center"
+    justify="center"
+  >
+    <Box alignSelf="center" direction="row" gap="small">
+      <Dot color={color} delay="-0.32s" />
```

Comment:
Maybe a comment talking about how these delays relate to the overall animation length/effect?